### PR TITLE
Jb51522

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: share/qtcreator/sfdk/modules/50-testing-mb2.json [hunk 1]
```diff
@@ -18,14 +18,14 @@
             "name": "deploy",
             "trSynopsis": "{--pkcon|--rsync|--sdk|--zypper|--manual} [--all] [--debug] [--] [[+|-]<pattern>...]",
             "trBrief": "Deploy to a device",
-            "configOptions": ["device!", "output-dir", "output-prefix", "target!", "[no-]task"],
+            "configOptions": ["*", "device!", "-[no-]fix-version", "-[no-]search-output-dir", "target!"],
```
Comment (vigejolla) [inline share/qtcreator/sfdk/modules/50-testing-mb2.json:21]: There's a lot more happening here than just adding the [no-]snapshot option. What does the asterisk do here? Why are you adding -[no-]fix-version and -[no-]search-output-dir? And removing output-dir, output-prefix and [no-]task?

Comment (martyone) [reply share/qtcreator/sfdk/modules/50-testing-mb2.json:21]: "*" - use all module options, "-" prefix - exclude, "!" suffix - mandatory

So this is just switching from the "start with none, then add" style to "start with all, then exclude" style.

Comment (vigejolla) [reply share/qtcreator/sfdk/modules/50-testing-mb2.json:21]: Of course...

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**